arches/app/media/js/views/components/cards/photo-gallery-card.js

```diff
@@ -49,6 +49,16 @@ define([
 
         this.fileListNodeId = getfileListNode();
 
+        this.maxFilesize = ko.computed(function(){
+            var mfs = "Missing maxFilesize"
+            self.card.widgets().forEach(function(widget){
+                if (widget.node_id() === self.fileListNodeId) {
+                    mfs = widget.config.maxFilesize();
+                }
+            });
+            return mfs;
```

Comment:
What should we show where there is no max file size? With this PR we show:

![Screenshot 2024-03-08 at 4 14 05 PM](https://github.com/archesproject/arches/assets/38668450/f20c8ef7-02d7-4c5c-a436-bfa3937c5208)

Reply:
..thanks @jacobtylerwalls!  

If undefined, let's inform the user.. Suggestion below..